src/backend/src/utils/google-integration.utils.ts

```diff
@@ -168,3 +172,172 @@ export const downloadImageFile = async (fileId: string) => {
     throw error;
   }
 };
+
+/**
+ * Creates a new google calendar on the NER google calendar
+ * @param name
+ * @returns the calendar id
+ */
+export const createCalendar = async (name: string) => {
+  if (process.env.NODE_ENV !== 'production') return;
+  try {
+    oauth2Client.setCredentials({
+      refresh_token: CALENDAR_REFRESH_TOKEN
+    });
+    const calendar = google.calendar({ version: 'v3', auth: oauth2Client });
+    const createdCalendar = await calendar.calendars.insert({
+      requestBody: { summary: `NER ${name} Meetings`, description: `A Team Type Within NER` }
+    });
+
+    return createdCalendar.data.id;
+  } catch (error: unknown) {
+    throw error;
+  }
+};
+
+/**
+ * Creates A Google Calendar Event on the NER Google Calendar
+ * @param members required and optional members
+ * @param calendarId the id of the calendar to add the event
+ * @param dateScheduled
+ * @param isInPerson
+ * @param zoomLink
+ * @param location
+ * @param meetingTimes
+ * @param wbsElement
+ * @returns the id of the calendar event
+ */
+export const createCalendarEvent = async (
+  calendarId: string | null,
+  memberIds: string[],
+  dateScheduled: Date,
+  isInPerson: boolean,
+  zoomLink: string | null,
+  location: string | null,
+  meetingTimes: number[],
+  wbsElement: WBS_Element
+) => {
+  if (process.env.NODE_ENV !== 'production') return;
+  if (!calendarId) throw Error('no calendar id provided');
```

Comment:
So that you dont have to do this, why don't you check first if there is a calendar and make one if there isnt

That way you don't have to check for it in this method